socialwarehouse/geo/models/address.py

```
@@ -281,6 +294,280 @@ def populate_foreign_keys(self):
+        field = f"{boundary_type}_geoid"
+
+        for abp in rows:
+            geoid = getattr(abp, field, None)
+            if not geoid:
+                continue
+
+            plan = None
+            if abp.redistricting_plan_id is not None:
+                try:
+                    plan = abp.redistricting_plan
+                except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)